.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,36 @@
+name: PR-Preview
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  REACT_APP_NETWORK_IDS: "100,10200"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
+
+      - name: Set env
+        run: echo "PUBLIC_URL=./" >> $GITHUB_ENV
```

Comment:
Why not put this directly in the top-level `env:`? Also, might be nice to document why this is necessary as it's not immediately obvious to me.

Reply:
now solved 🙌